src/Blocks/BlockIce.h

```diff
@@ -31,6 +31,44 @@ class cBlockIceHandler :
 		}
 	}
 
+	virtual void OnUpdate(
+		cChunkInterface & a_ChunkInterface,
+		cWorldInterface & a_WorldInterface,
+		cBlockPluginInterface & a_PluginInterface,
+		cChunk & a_Chunk,
+		const Vector3i a_RelPos
+	) const override
+	{
+		if (a_WorldInterface.GetDimension() == dimNether)
+		{
+			a_Chunk.SetBlock(a_RelPos, E_BLOCK_AIR, 0);
+		}
+		// Artificial light on any of the surrounding block > 11 leads to melting ice
+		bool Smelt = false;
+		Vector3i Pos;
+		cChunk *Chunk;
+
+		Vector3i Coords[7] =
+			{{1, 0, 0}, {-1, 0, 0},
+			{0, 1, 0}, {0, -1, 0},
+			{0, 0, 1}, {0, 0, -1},
+			{0, 0, 0}};
```

Comment:
by the way is 0,0,0 needed?

Reply:
It did show some strange behaviour when I didn't add it. The wiki says only surrounding blocks but that didn't melt the blocks like in vanilla

Reply:
could be the lighting being invalid perhaps? does sound strange.

Reply:
Maybe. It just melted if the light source block was at the same y coords. Doing some more research tomorrow